spec.bs

```diff
@@ -277,33 +278,45 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
   </dd>
 </dl>
 
+<div algorithm>
```

Comment:
Their declaration order is opposite their definition order; it'd be slightly nicer if they were matching. (I clicked on the first one in the IDL block and was surprised I had to scroll up, instead of down, to see the second one.)